pallets/gear-builtin/src/benchmarking.rs

```diff
@@ -254,6 +254,42 @@ benchmarks! {
         let standard = <ark_bls12_381::g2::Config as SWCurveConfig>::mul_projective(&base, &bigint);
         assert_eq!(standard, result.0);
     }
+
+    bls12_381_aggregate_g1 {
+        let c in 1 .. 1_000;
+
+        let count = c as usize;
+
+        let mut rng = ark_std::test_rng();
+
+        let points = (0..count).map(|_| G1::rand(&mut rng)).collect::<Vec<_>>();
+        let ark_points: ArkScale<Vec<G1>> = points.clone().into();
+        let encoded_points = ark_points.encode();
+
+        let mut _result = Err(0);
+    }: {
+        _result = gear_runtime_interface::gear_bls_12_381::aggregate_g1(&encoded_points);
+    } verify {
+        assert!(
+            matches!(_result, Ok(result) if ArkScale::<G1>::decode(&mut &result[..]).is_ok())
+        );
+    }
+
+    bls12_381_map_to_g2affine {
+        let c in 0 .. MAX_PAYLOAD_SIZE as u32;
+
+        let message = vec![1u8; c as usize];
+
+        let mut _result = Err(0);
+    }: {
```

Comment:
Better to use per_kb here like in other places. Also using per kb reduces weight round fluctuation. For example now you have 565 per byte and for kb it can be 565921.

Reply:
create new issue for that - https://github.com/gear-tech/gear/issues/4013